### ISSUE TITLE
preload function doesn't call resolve

### ISSUE THREAD

**Issue by daesign-dev** [zyra/ionic-image-loader#175]
**image-loader.ts**
```
if (this.shouldIndex) {
                this.addFileToIndex(file).then(() => {
                  this.getCachedImagePath(currentItem.imageUrl).then((localUrl) => {
                    currentItem.resolve(localUrl);
                    resolve();
                    done();
                    this.maintainCacheSize();
                  });
                });
}
```

If we don't want to limit cache size or duration, resolve() never call. Can we add :

```
if (this.shouldIndex) {
                this.addFileToIndex(file).then(() => {
                  this.getCachedImagePath(currentItem.imageUrl).then((localUrl) => {
                    currentItem.resolve(localUrl);
                    resolve();
                    done();
                    this.maintainCacheSize();
                  });
                });
}

// Add this ?
else
{
     currentItem.resolve(localUrl);
     resolve();
     done();
}
```

**Comment by eduardoRoth** [comment]
@daesign-dev this is already fixed in PR https://github.com/zyra/ionic-image-loader/pull/171

We're waiting for @ihadeed to review it and merge to master branch.

**Comment by danielsogl** [comment]
Closes with #171
